lib/plug/conn/query.ex

```diff
@@ -201,8 +201,8 @@ defmodule Plug.Conn.Query do
     [field, ?=|encode_value(value, encoder)]
   end
 
-  defp encode_dict(dict, parent_field, encoder) do
-    prune Enum.flat_map dict, fn
+  defp encode_map(map, parent_field, encoder) do
```

Comment:
This function is used by both map and keywords lists. So we should keep it as encode_dict OR call it encode_kw. :)